src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        Colors randomColor = ColorSupplier.getRandomColor();

```

Comment:
Make `getRandomColor` a non-static method and `ColorSupplier` instance a constant.